db/migrations/20181210091511-createSlackAutomationTable.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => queryInterface.createTable('slackAutomation', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    channelId: {
```

Comment:
@IAMOTZ, for reporting on the UI, will we have a way to change the channelId to the channel name? Otherwise the information might not mean much to the end user

Reply:
Yes, we would find a way to convert the `channelId` to the `channelName` on the front end.
This is because slack channel names can change at any time but the `channelIds` are constant, so we can always trust the `channelId` to refer to the exact channel.

Reply:
Thanks, that makes sense...